site/src/env.d.ts

```diff
@@ -1 +1,2 @@
+/// <reference path="../.astro/types.d.ts" />
```

Comment:
Not sure what's going on with this but it kept adding itself and given it's only types seems harmless